server/chat/backend/agent/utils/llm_usage_tracker.py

```
@@ -331,7 +331,19 @@ def store_usage(cls, usage: LLMUsage) -> bool:
-                
+
+                # Report usage to marketplace metering (no-op unless hooks are configured)
+                try:
+                    from utils.hooks import get_hook
+                    get_hook("report_usage")(usage.org_id, usage.estimated_cost, {
+                        "user_id": usage.user_id,
+                        "model": usage.model_name,
+                        "input_tokens": usage.input_tokens,
+                        "output_tokens": usage.output_tokens,
+                    })
+                except Exception as hook_err:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)